docs/validators/setup/priority_mempool.md

```diff
@@ -0,0 +1,50 @@
+<!--
+order: 5
+-->
+
+# Prioritized Mempool
+
+Learn about using the prioritized Tendermint mempool.{synopsis}
+
+The mempool holds uncommitted transactions, which are not yet included in a block.
+The default mempool for Tendermint blockchains follows a first-in-first-out (FIFO) principle,
+which means the ordering of transactions depends solely on the order in which they arrived at the node.
+
+Starting with [Tendermint v0.35](https://github.com/tendermint/tendermint/blob/v0.35.0/CHANGELOG.md)
+(has also been backported to [v0.34.20](https://github.com/tendermint/tendermint/blob/17c94bb0dcb354c57f49cdcd1e62f4742752c803/UPGRADING.md?plain=1#L54))
+it is possible to use a prioritized mempool implementation.
+This allows validators to choose transactions based on the associated fees or other incentive mechanisms.
+It is achieved by passing a `priority` field with each [`CheckTx` response](https://github.com/tendermint/tendermint/blob/17c94bb0dcb354c57f49cdcd1e62f4742752c803/proto/tendermint/abci/types.proto#L234),
+which is run on any transaction trying to enter the mempool.
+The current Cosmos SDK implementation allows the application layer to define a function of type [`TxFeeChecker`](https://github.com/cosmos/cosmos-sdk/blob/37a9bc3bb67bd82d4493d2d86f8cd31c0e768880/x/auth/ante/fee.go#L13),
```

Comment:
@MalteHerrmann What is implemented in the Evmos case? Do we still need to implement a `TxFeeChecker` function that calculates the priority based on the Priority tip?

Reply:
I'll leave the Slack conversation here for reference: https://evmos.slack.com/archives/C038HV08132/p1669974868743109